include/cucumber-cpp/internal/step/StepManager.hpp

```diff
@@ -183,6 +183,7 @@ class CUCUMBER_CPP_EXPORT StepManager {
 public:
     static step_id_type addStep(boost::shared_ptr<StepInfo> stepInfo);
     static MatchResult stepMatches(const std::string &stepDescription);
+    static std::ptrdiff_t codePointsFromStart(const std::string &expression, size_t index);
```

Comment:
commit msg: To me "codepoint position" or "codepoint offset" would sound more natural.